main.js

```diff
@@ -3,15 +3,45 @@ The DOMContentLoaded event gets fired by the browser after the dom is completely
 All html rendered, all images loaded, and, most importantly, all javascript files are loaded
 so its basically "wait till everything is good to go, then run this function". */
 
+const getMenu = () => {
+    // Selects the form
+    let food = document.getElementById('food');
+    // Select the Title Heading
+    let heading = document.querySelector('.menu__heading');
+    // Display Food Type
+    let displayFood = document.querySelector('.showtext');
+    // Identify the Type of Menu
+    let selectedValue = food.value;
+    // Create a heading depending of the type
+    if (selectedValue === 'breakfast') {
+        heading.innerHTML = 'You have created a Breakfast Menu';
+    } else if (selectedValue === 'lunch') {
+        heading.innerHTML = 'You have created a Lunch Menu';
+    } else if (selectedValue === 'dinner') {
+        heading.innerHTML = 'You have created a Dinner Menu';
+    }
+
+};
+
+
 document.addEventListener("DOMContentLoaded", function(){
```

Comment:
Our entire scrip should goes inside of 

`document.addEventListener("DOMContentLoaded", function(){`

});

Reply:
I take that back not inside but below my script as my script is the "loading" script

Reply:
when we put the drop down menu function after your script, it doesn't work I think. let me test a bit.